tests/tests/t01-inloggen-basisgebruiker.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { expect, test } from '@playwright/test';
+
+import { goToPageAndAcceptCookies } from '../helpers/go-to-page-and-accept-cookies';
+import { loginOnderwijsHetArchief } from '../helpers/login-onderwijs-het-archief';
+
+/**
+ * New: https://docs.google.com/spreadsheets/d/1IvhK0v0HSntCwTcXiFseHargwwWwpoCkDMjmMehaDMA/edit#gid=0
+ */
+
+test('T01: Inloggen basisgebruiker', async ({ page, context }) => {
+	await goToPageAndAcceptCookies(
+		page,
+		process.env.TEST_CLIENT_ENDPOINT as string,
+		// INT
+		//'Audiovisuele beeldbank voor leerkrachten | Het Archief voor Onderwijs'
+		// QAS
+		'Audiovisuele beeldbank | Het Archief voor Onderwijs'
+	);
+
+	await loginOnderwijsHetArchief(
+		page,
+		process.env.TEST_CLIENT_ENDPOINT as string,
+		process.env.TEST_BASIS_GEBRUIKER_USER as string,
+		process.env.TEST_BASIS_GEBRUIKER_PASS as string
+	);
+
+	// // Check navbar exists
```

Comment:
bij iedere stap moet je nog checken of het werkelijk werkt of niet
stel bovenstaande functie faalt en redirect naar de uitgelogde home pagina, dan slaagt de alsnog

best nog een test toevoegen die checks dat de gebruiker zijn naam rechts bovenaan ergens in beeld is.

Alle vermeldingen in de kolom onder E15 zijn voor dit soort "checks"

Dus de test mag niet gewoon alles met zijn ogen toe uitvoeren. Hij moet ook na iedere stap checken of de actie het gewenste resultaat heeft indien er een "verwacht resultaat" is ingevuld onder E15